pootle/apps/virtualfolder/urls.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.conf.urls import include, url
+
+from pootle_store.urls import unit_xhr_urlpatterns
+
+from .views import get_vfolder_units, VFolderTPTranslateView
+
+
+vfolder_urlpatterns = [
+    # TP Translate
+    url(r'(?P<language_code>[^/]*)/'
```

Comment:
Why not include here the `++vfolder/vfname`? That way it should be easier to generate this URLs, for example in https://github.com/translate/pootle/pull/5221/commits/79b56314f997cf934106394928baa65b111a59e6#diff-2595359c0c549cb2373dc7ec23dc8cb2R415